src/components/AvatarWithIndicator.tsx

```diff
@@ -11,7 +11,10 @@ import Tooltip from './Tooltip';
 
 type AvatarWithIndicatorProps = {
     /** URL for the avatar */
-    source: UserUtils.AvatarSource;
+    source?: UserUtils.AvatarSource;
+
+    /** account id if it's user avatar */
+    accountID?: number;
 
```

Comment:
Adding a comment to check https://github.com/Expensify/App/issues/40991 if it's something we can fix here

Reply:
followed up with a comment and I will spend some time today digging into this bug